tests/Tests/500__jupyterhub/test-data/rhods-users-empty.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: rhods-users
+users:
+  - nobodyhere
```

Comment:
Why not use the ldap users preloaded by Jenkins? It would make things easier (ldap-userX, ldap-adminX, ldap-noaccessX)